### PR TITLE
Revised data access rights vocabulary - please review

### PR DESCRIPTION
**Pull request by dxwell**
Changed vocab name to data-access-rights
Changed prefix to da:
Added skok:definition to Collections
Removed dct:created and dct:modified from concepts
Simplified definition for the ConceptScheme
Simplified definitions for Concepts

### CODE REVIEW

File: vocabularies/data-access-rights.ttl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+da:embargoed a skos:Concept ;
+    skos:prefLabel "Embargoed access"@en ;
+    skos:definition "Embargoed access refers to a resource that is metadata only access until released for open access on a certain date. Embargoes can be required by regulation or policies, or set by the author."@en ;
+    skos:inScheme da:conceptScheme ;
+    skos:topConceptOf da:conceptScheme .
+
+da:metadata-only a skos:Concept ;
+    skos:prefLabel "Metadata only access"@en ;
+    skos:definition "Metadata only access refers to a resource in which access is limited to metadata only. The metadata is open data but acccess to the resource itself is controlled. Access to the resource requires granting of access rights."@en ;
```
Comment (KellyVance) [inline vocabularies/data-access-rights.ttl]: Metadata only access refers to a resource in which access is limited to metadata only i.e. only metadata is accessible prior to a specified open-file date for the resource. Access to the resource requires granting of access rights.

_rewording and remove typo from access_

File: vocabularies/data-access-rights.ttl [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    dct:description "Data access rights control how users and systems access a data resource."@en ;
+    skos:historyNote "This vocabulary is taken from the COAR Controlled Vocabularies Interest Group (http://vocabularies.coar-repositories.org/documentation/access_rights/) but is redelivered as that vocabulary isn't well presented online."@en ;
+    dct:source <http://vocabularies.coar-repositories.org/documentation/access_rights/> ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:created "2019-04-03"^^xsd:dateTime ;
+    dct:modified "2019-09-10"^^xsd:dateTime .
+
+da:embargoed a skos:Concept ;
+    skos:prefLabel "Embargoed access"@en ;
+    skos:definition "Embargoed access refers to a resource that is metadata only access until released for open access on a certain date. Embargoes can be required by regulation or policies, or set by the author."@en ;
```
Comment (KellyVance) [inline vocabularies/data-access-rights.ttl]: Embargoed access refers to a resource accessible as metadata only, prior to open-access release on a specified date. 

_wording_

File: vocabularies/data-access-rights.ttl [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+da:open a skos:Concept ;
+    skos:prefLabel "Open access"@en ;
+    skos:altLabel "Open file"@en ;
+    skos:definition "Open access refers to a resource that is immediately and permanently online, and free for all on the Web, without financial and technical barriers."@en ;
+    skos:inScheme da:conceptScheme ;
+    skos:topConceptOf da:conceptScheme .
+
+da:restricted a skos:Concept ;
+    skos:prefLabel "Restricted access"@en ;
+    skos:definition "Restricted access refers to a resource that is available in a system but with some type of restriction for full open access. Access is limited to personnel who need the resources to do their work."@en ;
```
Comment (KellyVance) [inline vocabularies/data-access-rights.ttl]: Restricted access refers to a resource that is stored in a system but is not freely accessible. Access is limited to specific personnel or user groups.

_wording_

File: vocabularies/data-access-rights.ttl [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    skos:inScheme da:conceptScheme ;
+    skos:topConceptOf da:conceptScheme .
+
+da:open-access-rights a skos:Collection ;
+    skos:prefLabel "Open data access rights"@en ;
+    skos:definition "Data that is non-sensitive, freely available, easily discovered and accessed, and published in ways and with licences that allow easy reuse."@en ;
```
Comment (KellyVance) [inline vocabularies/data-access-rights.ttl]: Data that is non-sensitive, freely available, discoverable, accessible, and published under with licences that allow reuse.

_wording_

File: vocabularies/data-access-rights.ttl (hunk 3 above)
Comment (dxwell) [reply vocabularies/data-access-rights.ttl]: Changed verbatim

**Pull request merged**